retrodeck.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Steam Deck SD path: /run/media/mmcblk0p1
```

Comment:
flathub is more than just steam deck. it actually predates steam deck.

Reply:
I know, however my software is tailored and it's supported only on Steam Deck, for other platforms there are alternatives like Batocera or RetroPie for example.

Reply:
> Sure, but again, Flathub is not about SteamDeck. It needs to be done in a way that either works everywhere, or appdata needs to clearly state what is it for and the app should show an error and gracefully exit instead of operating with a bunch of incorrect assumptions.

How can I state this?

Reply:
Added in the appdata

Reply:
This is still something that has to be detected also by your script, so it exits when it detects it's not running on SteamDeck instead of continuing.